tests/eoc_test.cpp

```diff
@@ -246,6 +246,15 @@ TEST_CASE( "EOC_transform_radius", "[eoc][timed_event]" )
     calendar::turn += 2_seconds;
     get_timed_events().process();
     check_ter_in_radius( start, eoc_range, t_grass );
+    CHECK( "Cleaning up avatar's pos and location after successful test..." );
```

Comment:
Are you meaning to use something other than CHECK? This is an odd use of it.

Reply:
In case it fails due to future changes, this was intended to be somewhat informative output. (I meant to use CHECK_NOFAIL, oops)

Reply:
I'm marking this resolved because we seem to have cleared that up and the change to CHECK_NOFAIL is only relevant to style, but *please do* open another review/comment if you have any concern. It is greatly appreciated.